lib/tasks/wb_extra.rake

```diff
@@ -0,0 +1,6 @@
+namespace :wb_extra do
```

Comment:
Please add a reference for this task to the global import task in `lib/tasks/db.rake`